evergreen/run-tests.sh

```diff
@@ -21,18 +21,12 @@ TOPOLOGY=${TOPOLOGY:-server}
 
 provision_ssl () {
   echo "SSL !"
-
+  uri_environment_variable_name=$1
   # Arguments for auth + SSL
   if [ "$AUTH" != "noauth" ] || [ "$TOPOLOGY" == "replica_set" ]; then
-    export MONGODB_URI="${MONGODB_URI}&ssl=true&sslVerifyCertificate=false"
-    if [ "$TOPOLOGY" == "sharded_cluster" ]; then
-      export MONGODB_URI_WITH_MULTIPLE_MONGOSES="${MONGODB_URI_WITH_MULTIPLE_MONGOSES}&ssl=true&sslVerifyCertificate=false"
-    fi
+    export $uri_environment_variable_name="${!uri_environment_variable_name}&ssl=true&sslVerifyCertificate=false"
```

Comment:
``${!uri_environment_variable_name}`` give us indirect expansion in bash. E.G.:
If `uri_environment_variable_name` is `MONGODB_URI`, and ``MONGOD_URI`` is ``mongodb://localhost`` then ``${!uri_environment_variable_name}`` expands to ``mongodb://localhost``

Reply:
Acknowledged